### ISSUE TITLE
New Machine Learning Question

### ISSUE THREAD

**Issue by MakendranG** [Ebazhanov/linkedin-skill-assessments-quizzes#4946]
Fill in the blanks: Two multivariate imputer techniques are the _____ imputer and the _____ imputer.

supervised, unsupervised
**iterative, KNN**
similarity, regressive
normalized, scaled

You are working on a regression model using the Keras library.  What method on the Model class do you use to train the model?

predict
compile
**fit**
get_weights

What is the goal of regularization in the K nearest neighbors algorithm?

normalizing the data points so they can be compared with each other
using a straight line model to make predictions based on training data
finding the slope of the line that represents the model
**making the decision boundaries more regular**

If there is no trend between two variables x and y, we say that there is a _____ connection between x and y.

linear
exponential
non-random
**random**

**Comment by MakendranG** [comment]
Hi @Ebazhanov

I have worked on this issue and raised a PR https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/pull/4945

Kindly merge it.
